### ISSUE TITLE
Use ~/.aws/credentials

### ISSUE THREAD

**Issue by kaihendry** [aws/amazon-ecs-cli#372]
Been staring at the aws-cli configure breakage and http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_CLI_Configuration.html and I still can't quite figure out how to use the credentials in `~/.aws/credentials`.

What am I missing?

**Comment by mattmccarty** [comment]
@kaihendry I opened a similar ticket yesterday. It has more to do with using assumed roles, but should still apply to your case. See my examples [in the ticket](https://github.com/aws/amazon-ecs-cli/issues/369). The issue was opened for v1.0.0 btw.

**Comment by PettitWesley** [comment]
@kaihendry Apologies lack of clarity on this in our public documentation. We have added a new flag `--aws-profile` (Env Var: `$AWS_PROFILE`) that allows you to reference any profile in `~/.aws/credentials`.

Let me know if you have any more questions about the configuration/credential related changes in v1.0!

**Comment by dbahlbeck** [comment]
I have some trouble using `--aws-profile`. The AWS profile I'm trying to use does not seem to work if it refers to credentials in a source profile:
```
[myprofile]
# this profile has an access key

[sandbox]
output = json
region = eu-west-1
role_arn = arn:aws:iam::123456789012:role/ROLE
source_profile = myprofile
```
It fails with the error: `credentials did not contain aws_access_key_id caused by: error when getting key of section 'sandbox': key 'aws_access_key_id' not exists"`

**Comment by PettitWesley** [comment]
@dbahlbeck, strong apologies for the delay in response. This is caused by the issue discussed in #369. It's been fixed with ECS CLI v1.1.0 which was release last week :)
